internal/build/pom.xml

```diff
@@ -569,6 +578,12 @@
                 <optional>true</optional>
             </dependency>
 
+            <dependency>
+                <groupId>org.moditect.jfrunit</groupId>
+                <artifactId>jfrunit-core</artifactId>
+                <version>1.0.0.Alpha2</version>
```

Comment:
move the version into `${dep.jfrunit.version}` ?

Reply:
There's only one module so the property doesn't seem to bring much value, but ok.